### ISSUE TITLE
[Issue]: Windows server / .Net-Framework 4.8.1

### ISSUE THREAD

**Issue by Tfreak586** [jellyfin/jellyfin-server-windows#119]
### Please describe your bug

Windows server 2019 and older cant use the new Jellyfin server. 
Because .Net-Framework 4.8.1 is required and windows does not allow to install this version.

### Reproduction Steps

Install newest server version.
Try to start its not working because of .Net is not version 4.8.1

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: Windows Server 2019
- Linux Kernel: -
- Virtualization: -
- Clients: -
- Browser: -
- FFmpeg Version: -
- Playback Method: -
- Hardware Acceleration: -
- GPU Model: -
- Plugins: -
- Reverse Proxy: -
- Base URL: -
- Networking: -
- Storage:-
```

### Jellyfin logs

```shell
No Logs because cant start.
Only window that open´s is "you need .NET 4.8.1"
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nielsvanvelzen** [comment]
What is the highest .NET framework version available for you?

**Comment by Tfreak586** [comment]
The highest version for windows server 2012, 2016, 2019 is 4.8.0
You can find this information here too:
https://learn.microsoft.com/de-de/dotnet/framework/migration-guide/versions-and-dependencies#net-framework-481

**Comment by nirwanah** [comment]
I'm also experiencing this issue. This is also affecting the Media Player users who will not be able to connect to Jellyfin Server as Jellyfin Media Player v1.10.0 and newer requires Jellyfin Server 10.9.1 or later to work. 

So the last compatible option for Windows Server 2019 users seems to be:
Player: 1.9.1
Server: 10.8.13

**Comment by DrDagless** [comment]
Same problem with my server. Looks like it's been fixed by #121 so hopefully gets pushed to an update soon.

**Comment by anthonylavado** [comment]
There are issues with the way we currently handle the build when it comes to either of those fixes, so the next update from today is going out "as-is" until we can fix it properly.
